README.md

```diff
@@ -94,23 +90,26 @@ Logging can be done in many ways. [This blog post](https://goldmann.pl/blog/2014
 
 Sometimes you need to customize the application server configuration. There are many ways to do it and [this blog post](https://goldmann.pl/blog/2014/07/23/customizing-the-configuration-of-the-wildfly-docker-image/) tries to summarize it.
 
-## Extending the image
+## Extending the image with the management console
 
-To be able to create a management user to access the administration console create a Dockerfile with the following content
+To be able to create an admin user to access the management console create a `Dockerfile` with the following content
 
     FROM quay.io/wildfly/wildfly
-    RUN /opt/jboss/wildfly/bin/add-user.sh admin Admin#70365 --silent
+
+    RUN --mount=type=secret,id=ADMIN_PASSWORD,env=ADMIN_PASSWORD,required=true \
+        $JBOSS_HOME/bin/add-user.sh admin ${ADMIN_PASSWORD} --silent
```

Comment:
FYI we should not be hard coding usernames, using admin is also a bad practice to be avoided.

Reply:
@darranl Thanks for the reminder, I've updated the readme to mount both username and password.